joblib/backports.py

```diff
@@ -0,0 +1,25 @@
+"""
+Backports of fixes for joblib dependencies
+"""
+
+from distutils.version import LooseVersion
+
+try:
+    import numpy as np
+
+    def make_memmap(filename, dtype='uint8', mode='r+', offset=0,
```

Comment:
I don't mind either way. In this case this is to have uniform signature in the case where numpy is not available ... Not a great reason but there you go.